projects/go-lib/src/lib/components/go-action-sheet/go-panel/go-panel.component.scss

```diff
@@ -72,6 +72,11 @@ $base-medium: #767676 !default;
   &--header {
     color: $base-medium;
   }
+
+  &--disabled {
+    opacity: 0.5;
```

Comment:
If you look at the action sheet, if we disable the first panel then its opacity is affected causing the element behind it to be shown and look weird. I'm not sure we can use opacity here.

An alternative would be to change the `color` property to use an `rgba` value with.

```scss
color: transparentize($theme-light-color, .5);